docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<p>Please remove the comma as it is unnessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>You could just move the comma forward so it makes more sense: <em>We begin by importing STUMPY </em><strong><em>and</em></strong><em> some other relevant packages for loading</em><strong><em>,</em></strong><em> and analyzing the data.</em></p>

Reply:
Resolved